compiler/resolution/functionResolution.cpp

```diff
@@ -2064,6 +2069,55 @@ FnSymbol* tryResolveCall(CallExpr* call) {
   return resolveNormalCall(call, true);
 }
 
+static bool resolveTypeComparisonCall(CallExpr* call) {
+
+  if (UnresolvedSymExpr* urse = toUnresolvedSymExpr(call->baseExpr)) {
+    if (call->numActuals() == 2) {
+      const char* name = urse->unresolved;
+
+      if (name[0] == '<' || name[0] == '>') {
+
```

Comment:
How about using comparisons like `name == astrSlt` `name == astrSge` etc. following the lead of astrSdot et al.?